core/src/test/java/nl/nn/adapterframework/receivers/ReceiverTest.java

```diff
@@ -563,4 +564,42 @@ public void startReceiver() throws Exception {
 		assertEquals(RunState.STOPPED, receiver.getRunState());
 		assertEquals(RunState.STOPPED, adapter.getRunState());
 	}
+
+	@Test
+	public void processLargeResult() throws Exception {
```

Comment:
I'd suggest a name which reflects the special situation you're testing. Such as: `processTooLargeMessageSize`, which is a bit exaggerating, but ok.

Reply:
`testResultLargerThanMaxCommentSize`?